hosted/README.md

```diff
@@ -61,40 +59,36 @@
 ## `p0_importing_test.go`
 
 - **Describe:** P0Importing
-    - **It:** should successfully import the cluster
+    - **It:** should successfully import the cluster & add, delete, scale nodepool
```

Comment:
Can you also add the automation script that updates the readme?

Reply:
Done